deployments/local-scripts/relay-messages-millau-to-rialto-entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -xeu
+
+RUST_LOG=bridge=debug \
+./target/debug/substrate-relay relay-messages millau-to-rialto \
+	--lane 00000000 \
+	--millau-host localhost \
+	--millau-port 9945 \
+	--millau-signer //Eve \
```

Comment:
The other scripts that are part of `local-scripts` don't funds for `Eve` since they run as `--dev` nodes, so I don't think this will work. Can you either a) check if this works or b) use a different account

Reply:
Should i set Alice or Bob ?

Reply:
I would do `Bob` since `Alice` is used by the header sync scripts